app/_src/gateway/admin-api/index_latest.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Kong Gateway Admin API
+no_version: true
+---
+
+
+<!-- vale off -->
+
+{{site.base_gateway}} comes with an **internal** RESTful Admin API for administration purposes.
+ Requests to the Admin API can be sent to any node in the cluster, and Kong will
+ keep the configuration consistent across all nodes.
+
+ - `8001` is the default port on which the Admin API listens.
+ - `8444` is the default port for HTTPS traffic to the Admin API.
+
+ This API is designed for internal use and provides full control over Kong, so
+ care should be taken when setting up Kong environments to avoid undue public
+ exposure of this API. See [this document][secure-admin-api] for a discussion
+ of methods to secure the Admin API.
+
+### Documentation
+
+
+| Spec | Insomnia link |
+|-------|---------------|
+| [Enterprise beta API spec](/gateway/api/admin-ee/latest/) |<a href="https://insomnia.rest/run/?label=Kong%20Gateway%20Enterprise%203.4&uri=https%3A%2F%2Fraw.githubusercontent.com%2FKong%2Fdocs.konghq.com%2Fmain%2Fapi-specs%2FGateway-EE%2F3.4%2Fkong-ee-3.4.json" target="_blank"><img src="https://insomnia.rest/images/run.svg" alt="Run in Insomnia"></a>  |
+|  [Open source beta API spec](/gateway/api/admin-oss/latest/) |  <a href="https://insomnia.rest/run/?label=Kong%20Gateway%20Open%20Source%203.4&uri=https%3A%2F%2Fraw.githubusercontent.com%2FKong%2Fdocs.konghq.com%2Fmain%2Fapi-specs%2FGateway-OSS%2F3.4%2Fkong-oss-3.4.json" target="_blank"><img src="https://insomnia.rest/images/run.svg" alt="Run in Insomnia"></a>|
+
+
+|        |   Endpoints        | |
```

Comment:
```suggestion
| Endpoints | | |
```
Moving the label to the first column. In context with the specs table above it, it looks odd to have the text centered:
![Screenshot 2024-01-31 at 3 19 18 PM](https://github.com/Kong/docs.konghq.com/assets/54370747/e2652b32-c116-478a-b463-d1b971ff60a9)

Vs aligned to the left on both:
![Screenshot 2024-01-31 at 3 19 11 PM](https://github.com/Kong/docs.konghq.com/assets/54370747/4cdbf71a-9ba4-45f9-a454-3c5ac8244e35)